lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -687,7 +675,17 @@ void EmitterBase::emitTextWithSubstitutions(
         operandEmitter(op->getOperand(operandNo));
       else if ((operandNo - op->getNumOperands()) < numSymOps) {
         unsigned symOpNum = operandNo - op->getNumOperands();
-        os << symOps[symOpNum];
+        auto sym = symAttrs[symOpNum];
+        SmallString<12> symVerilogName;
```

Comment:
As Darth says, please change this to be a StringRef, there is no need to copy the stringref into a vector before printing it.